tests/foreman/installer/test_infoblox.py

```diff
@@ -15,183 +17,182 @@
 :Upstream: No
 """
 import pytest
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_set_dns_provider():
-    """Check Infoblox DNS plugin is set as provider
-
-    :id: 23f76fa8-79bb-11e6-a3d4-68f72889dc7f
-
-    :Steps: Set infoblox as dns provider with options
-        --foreman-proxy-dns=true
-        --foreman-proxy-plugin-provider=infoblox
-        --enable-foreman-proxy-plugin-dns-infoblox
-        --foreman-proxy-plugin-dns-infoblox-dns-server=<ip>
-        --foreman-proxy-plugin-dns-infoblox-username=<username>
-        --foreman-proxy-plugin-dns-infoblox-password=<password>
-
-    :expectedresults: Check inflobox is set as DNS provider
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_set_dhcp_provider():
-    """Check Infoblox DHCP plugin is set as provider
-
-    :id: 40783976-7e68-11e6-b728-68f72889dc7f
-
-    :Steps: Set infoblox as dhcp provider with options
-        --foreman-proxy-dhcp=true
-        --foreman-proxy-plugin-dhcp-provider=infoblox
-        --enable-foreman-proxy-plugin-dhcp-infoblox
-        --foreman-proxy-plugin-dhcp-infoblox-dhcp-server=<ip>
-        --foreman-proxy-plugin-dhcp-infoblox-username=<username>
-        --foreman-proxy-plugin-dhcp-infoblox-password=<password>
-
-    :expectedresults: Check inflobox is set as DHCP provider
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_update_dns_appliance_credentials():
-    """Check infoblox appliance credentials are updated
-
-    :id: 2e84a8b4-79b6-11e6-8bf8-68f72889dc7f
-
-    :Steps: Pass appliance credentials via installer options
-        --foreman-proxy-plugin-dns-infoblox-username=<username>
-        --foreman-proxy-plugin-dns-infoblox-password=<password>
-
-    :expectedresults: config/dns_infoblox.yml should be updated with
-        infoblox_hostname, username & password
```

Comment:
can you check if this config exists if we enable this plugin and add assertions to validate it?

Reply:
Yes, missed this. Assertions added.